lib/compress/zstd_compress.c

```diff
@@ -116,15 +114,12 @@ struct ZSTD_CCtx_s {
     U32    frameEnded;
 
     /* Dictionary */
-    ZSTD_dictMode_e dictMode; /* select restricting dictionary to "rawContent" or "fullDict" only */
-    U32 dictContentByRef;
     ZSTD_CDict* cdictLocal;
     const ZSTD_CDict* cdict;
     const void* prefix;
```

Comment:
minor detail : 
I just realised that `prefix`, `prefixSize` and `pledgedSrcSizePlusOne` share the same property : 
they are valid only once, for the next compression job.
After that, they must be reset to `0`.

In the future, it might be useful to join then into some identical "category".